buildSrc/src/main/kotlin/JvmConfig.kt

```diff
@@ -15,41 +16,43 @@ fun Project.configureJvm() {
         else -> 8
     }
 
-    val jvm = kotlin.jvm()
+    kotlin {
+        jvm()
 
-    kotlin.sourceSets {
-        val jvmMain by getting {
-            dependencies {
-                if (jdk > 6) {
-                    api("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Versions.kotlin}")
-                }
-                if (jdk > 7) {
-                    api("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Versions.kotlin}")
-                    api("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:${Versions.coroutines}") {
-                        exclude(module = "kotlin-stdlib")
-                        exclude(module = "kotlin-stdlib-jvm")
-                        exclude(module = "kotlin-stdlib-jdk8")
-                        exclude(module = "kotlin-stdlib-jdk7")
+        sourceSets {
+            jvmMain {
+                dependencies {
+                    if (jdk > 6) {
```

Comment:
FYI: there is no need anymore to provide different versions of stdlib as everything now is available in just stdlib - which is applied by default.
the same for coroutines dependency.

So it's possible to simplify everything and just not add stdlib/coroutines in platform sourceSets, as `coroutines` are already added in `CommonConfig`